tox.ini

```diff
@@ -7,6 +7,7 @@ python =
   3.4: py34
   3.5: py35
   3.6: py36, lint, restlint
+  3.7: py37, lint, restlint
```

Comment:
Note-to-self; we should probably drop 3.4 and add 3.8.

Reply:
Do you want me to change this in this PR? lemme know

Reply:
I have a new PR coming that'll also do this. So leave it as is.